### ISSUE TITLE
stg_sources definition too large

### ISSUE THREAD

**Issue by belalaboabdo** [dbt-labs/dbt-project-evaluator#297]
### Describe the bug

dbt build for `models/staging/graph/stg_sources.sql` fails due to definition being too large. dbt project contains 20+ sources some of which with 900+ tables. View definition exceeds Snowflake limit.

### Steps to reproduce

Large source yml that exceeds Snowflake's memory limit for defining a view. Cannot include project's source yml files.

### Expected results

Model builds successfully.

### Actual results

```
Database Error in model stg_sources (models/staging/graph/stg_sources.sql)
--
090222 (42601): View definition too large.
compiled Code at target/run/dbt_project_evaluator/models/staging/graph/stg_sources.sql
```

### System information
```
packages:
  - package: dbt-labs/codegen
    version: 0.9.0

  - package: fivetran/fivetran_log
    version: 0.7.0
  
  - package: fivetran/jira
    version: [">=0.11.0", "<0.12.0"]

  - package: fivetran/github
    version: [">=0.6.0", "<0.7.0"]

  - package: entechlog/dbt_snow_mask
    version: 0.2.3

  - package: brooklyn-data/dbt_artifacts
    version: 2.2.2

  - package: dbt-labs/dbt_project_evaluator
    version: 0.3.0
```
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [X] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
1.3.0
```

**Comment by dave-connors-3** [comment]
hey @belalaboabdo ! This should be addressed and fixed in the latest release, version `0.4.1`. Can you upgrade your package version and let us know if that resolves the issue?

**Comment by belalaboabdo** [comment]
> hey @belalaboabdo ! This should be addressed and fixed in the latest release, version `0.4.1`. Can you upgrade your package version and let us know if that resolves the issue?

Thanks I'll give that a try

**Comment by belalaboabdo** [comment]
Upgrading the package to `0.4.1` resolved the issue thanks
